grafana-plugin/src/module.ts

```diff
@@ -1,15 +1,47 @@
 import { ComponentClass } from 'react';
 
-import { AppPlugin } from '@grafana/data';
+import { AppPlugin, PluginExtensionPoints } from '@grafana/data';
 
+import MobileAppConnection from 'containers/MobileAppConnection/MobileAppConnection';
 import PluginConfigPage from 'containers/PluginConfigPage/PluginConfigPage';
 import { GrafanaPluginRootPage } from 'plugin/GrafanaPluginRootPage';
+import { getGrafanaVersion } from 'plugin/GrafanaPluginRootPage.helpers';
 
 import { OnCallPluginConfigPageProps, OnCallPluginMetaJSONData } from './types';
 
-export const plugin = new AppPlugin<OnCallPluginMetaJSONData>().setRootPage(GrafanaPluginRootPage).addConfigPage({
+const plugin = new AppPlugin<OnCallPluginMetaJSONData>().setRootPage(GrafanaPluginRootPage).addConfigPage({
   title: 'Configuration',
   icon: 'cog',
   body: PluginConfigPage as unknown as ComponentClass<OnCallPluginConfigPageProps, unknown>,
   id: 'configuration',
 });
+
+if (isUseProfileExtensionPointEnabled()) {
+  // TODO Change this to use the PluginExtensionPoints enum once that is available in @grafana/data (>10.2.2)
+  const extensionPointId = 'grafana/user/profile/tab';
```

Comment:
this can _probably_ be updated to `PluginExtensionPoints.UserProfileTab `

Reply:
That was not on the PluginExtensionPoints last time I checked, I believe that package doesn't have tge change yet

Reply:
done now since it's available in the latest package